docs/static/robots.txt

```diff
@@ -1,2 +1,4 @@
+Sitemap: https://docs.yugabyte.com/sitemap.xml
 User-agent: *
-Disallow: /v1.0/
+Disallow: /latest/api/ysql/syntax_resources/commands/*.diagram
+Disallow: /latest/api/ysql/syntax_resources/commands/*.grammar
```

Comment:
@stevebang I think the file name patterns are `*.diagram.md` and `*.grammar.md`.
Also, the logical rule would be to only allow `grammar_diagrams.md` from the syntax_resources folder.
Can we do something like:
```
Disallow: /latest/api/ysql/syntax_resources/
Allow: /latest/api/ysql/syntax_resources/grammar_diagrams.md
```

Reply:
Based on our discussion, changed to:
```
Disallow: /latest/api/ysql/syntax_resources/
Allow: /latest/api/ysql/syntax_resources/grammar_diagrams/
```